wscript

```diff
@@ -741,6 +741,11 @@ video_output_features = [
         'desc': 'EGL helper functions',
         'deps': 'egl || rpi || egl-angle-win32 || egl-android',
         'func': check_true
+    }, {
+        'name': '--sixel',
+        'desc': 'Sixel',
+        'deps': 'gpl',
+        'func': check_pkg_config('libsixel', '>= 1.8'),
```

Comment:
Do you need features of libsixel version 1.8? if not, please use a minimum version which the code actually needs.

For instance, it's possible that the libsixel version which is available on common LTS distros (debtan stable, Ubuntu LTS, maybe some fedora variants, etc) is older than 1.8 but would still work fine with the code, then it's not very nice to exclude them for no real reason.

Reply:
I looked at the ffmpeg implementation and since it was 5 years older and commit message said libsixel 1.5, I've changed it here to be 1.5 as well. I tried building mpv inside ubuntu 16.04 docker but looks like the ffmpeg dev files installed by apt were too old and hence mpv refused to compile. The code should work fine since ffmpeg code was based on libsixel 1.5 but haven't tested it.